tests/hardhat/Comptroller/liquidateAccountTest.ts

```diff
@@ -342,4 +343,129 @@ describe("liquidateAccount", () => {
       );
     });
   });
+
+  describe("setForcedLiquidation", async () => {
+    it("fails if asset is not listed", async () => {
+      const someVToken = await smock.fake<VToken>("VToken");
+      await expect(comptroller.setForcedLiquidation(someVToken.address, true)).to.be.revertedWithCustomError(
+        comptroller,
+        "MarketNotListed",
+      );
+    });
+
+    it("fails if ACM does not allow the call", async () => {
+      accessControl.isAllowedToCall.returns(false);
+      await expect(comptroller.setForcedLiquidation(OMG.address, true)).to.be.revertedWithCustomError(
+        comptroller,
+        "Unauthorized",
+      );
+      accessControl.isAllowedToCall.returns(true);
+    });
+
+    it("sets forced liquidation", async () => {
+      await comptroller.setForcedLiquidation(OMG.address, true);
+      expect(await comptroller.isForcedLiquidationEnabled(OMG.address)).to.be.true;
+
+      await comptroller.setForcedLiquidation(OMG.address, false);
+      expect(await comptroller.isForcedLiquidationEnabled(OMG.address)).to.be.false;
+    });
+
+    it("emits IsForcedLiquidationEnabledUpdated event", async () => {
+      const tx1 = await comptroller.setForcedLiquidation(OMG.address, true);
+      await expect(tx1).to.emit(comptroller, "IsForcedLiquidationEnabledUpdated").withArgs(OMG.address, true);
+
+      const tx2 = await comptroller.setForcedLiquidation(OMG.address, false);
+      await expect(tx2).to.emit(comptroller, "IsForcedLiquidationEnabledUpdated").withArgs(OMG.address, false);
+    });
+  });
+
+  describe("preLiquidateHook", async () => {
+    let accounts: SignerWithAddress[];
+
+    before(async () => {
+      accounts = await ethers.getSigners();
+    });
+    const generalTests = () => {
```

Comment:
We don't need this function if we don't have the second block. I guess we could just put these test cases right into the "isForcedLiquidationEnabled == true" block.

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/305/commits/cbd9b18a99c4e1f92bf9404e88fceb8ebc36d55f